uefi/region_test.go

```diff
@@ -0,0 +1,61 @@
+package uefi
+
+import "testing"
+
+var regionTestcases = [...]struct {
+	in    *Region
```

Comment:
Is there a reason you made it a region pointer instead of just putting the region struct in?

Reply:
The pointer is the actual input for the methods being tested. Apart from that, no.

Should I change this?

Reply:
no reason to make it a pointer, just embed the struct imo